.github/workflows/config/spelling_allowlist.txt

```diff
@@ -297,6 +297,7 @@ prepend
 preprocessor
 probability
 programmatically
+prx
```

Comment:
The spelling allowlist should only contain words that can be found in a dictionary. For code snippets, please use backticks (`) to mark them as such and make the spellcheck happy.